service_configuration_lib/spark_config.py

```diff
@@ -397,6 +395,8 @@ def __init__(self):
             ) = utils.load_spark_srv_conf()
         except Exception as e:
             log.error(f'Failed to load Spark srv configs: {e}')
+            # should fail because Spark config calculation depends on values in srv-configs
+            sys.exit(1)
```

Comment:
might be a little cleaner to throw an exception - `sys.exit()` outside of something like the driver for a script can be a little unexpected